core/templates/dev/head/components/OutcomeEditorDirective.js

```diff
@@ -63,6 +67,21 @@ oppia.directive('outcomeEditor', [function() {
           onExternalSave();
         });
 
+        $scope.getCurrentInteractionId = function() {
```

Comment:
I'm not sure why this and the following method are needed here; could you not just read the value of suppressWarnings() that was passed in?